rfcs/20211118-support-implicit-concat/README.md

```diff
@@ -0,0 +1,113 @@
+# RFC: Support of implicit concat
+
+## Introduction
+
+This document explores a way of supporting implicit concat. Implicit concat is a
+concat primitive which is done without any additional copies just by storing
+correspondent parts into right spots of final tensor. Such approach allows to
+save at least one memory pass over the final output. Sometimes it may be two or
+more.
+
+### Motivation
+
+Since the compute powers are only increasing without supporting proper memory
```

Comment:
Consider rewording: "Because compute power tends to increase more quickly than the memory bandwidth needed to support it, ..."

Reply:
Thank you for a nice suggestion!